circle/docker-release-image.sh

```diff
@@ -285,6 +285,24 @@ chart_update_image() {
   esac
 }
 
+chart_update_appVersion() {
```

Comment:
Thanks for the suggestion. I will add a ci script variable to disable the `appVersion` updates. It should work fine for our current use-cases.